packages/website/src/shared/lib/utilities/string.test.ts

```diff
@@ -0,0 +1,45 @@
+import { isSubstring } from "./string";
+
+describe("String Utilities", () => {
+  describe("isSubstring", () => {
+    it("handles without case-sensitivity", () => {
+      expect(isSubstring("", "")).toStrictEqual(true);
+
+      expect(isSubstring("ABC", "a")).toStrictEqual(true);
+      expect(isSubstring("ABC", "b")).toStrictEqual(true);
+      expect(isSubstring("ABC", "c")).toStrictEqual(true);
+
+      expect(isSubstring("ABC", "A")).toStrictEqual(true);
+      expect(isSubstring("ABC", "B")).toStrictEqual(true);
+      expect(isSubstring("ABC", "C")).toStrictEqual(true);
+
+      expect(isSubstring("ABC", "d")).toStrictEqual(false);
+      expect(isSubstring("ABC", "e")).toStrictEqual(false);
```

Comment:
test are meticulously and exhaustive, a good thing. nit. However, cases like "e' does not add to test suite, it is duplicate of 'd'

Reply:
Yeah, a lot of these tests are "duplicates" in the sense that the test the same thing. It doesn't add anything, but it's included for completeness and a way to give more than 1 example for a test case.